templates/Dataland/v2/index.html.twig

```diff
@@ -25,26 +25,48 @@
                         {% include 'Dataland/v2/status-badge.html.twig' %}
                         <h2> {{ dataset.title }} </h2>
                     </div>
-                    <div class="d-flex justify-content-between py-2">
-                        <div>
+                    {% if dataset.datasetsubmissionstatus != constant("App\\Entity\\DatasetSubmission::STATUS_UNSUBMITTED") %}
```

Comment:
Was thinking more along the lines of `if dataset.hasSubmittedSubmisson` or something like that.